### DISCUSSION TITLE
Age ranges for people_vaccinated

### DISCUSSION THREAD

**Discussion by savannalynhenderson** [owid/covid-19-data#2834] [Q&A] [answered]
Does this variable cover adults only (18 and up) or does it include children as young as 5? I'm specifically interested in this for Burkina Faso, Ethiopia, Kenya, Nigeria and Uganda.

**Comment by lucasrodes** [comment, marked as answer]
Hi @savannalynhenderson, thanks for your interest!
We source vaccination data for these countries from the WHO. From having a look at [their dashboard](https://data.who.int/dashboards/covid19/vaccines?n=o), we haven't found any reference to whether the numbers only include adults or not. Instead, they refer to "total population". Therefore, we've assumed that their figures include _all the population_.

**Reply by savannalynhenderson** [reply]
Thanks, @lucasrodes !
